blog/2023-06-23-newsletter.mdx

```diff
@@ -0,0 +1,164 @@
+
+---
+
+## Upcoming events
+
+[<Image img={scipybanner} alt="Banner image for SciPy2023"/>](https://www.scipy2023.scipy.org/)
+
+### SciPy 2023 Conda & Friends Sprint, July 15-16, Austin
+
+***Probably...*** *We have proposed a sprint for SciPy, but we haven't heard back. In the past the conference has approved our requests. So, until that happens, we are *tentatively* announcing that there will be a Conda & Friends sprint at SciPy, but we reserve the right to take it back if our foresight skills are weak.*
+
+> The conda community will again be meeting at [SciPy 2023](https://www.scipy2023.scipy.org/) in Austin, Texas for two days of collaborative work.
+>
+> [Sprints](https://www.scipy2023.scipy.org/sprints) are open to everyone who is interested in contributing to the various projects in the conda ecosystem (e.g. projects under the [conda](https://github.com/conda), [conda-incubator](https://github.com/conda-incubator), [mamba-org](https://github.com/mamba-org), [conda-forge](https://github.com/conda-forge), ... GitHub organizations), what we call "conda and friends".
+>
+> First and foremost, **the goal of conda community sprints is to grow the conda contributor community**. We mean "contributor" in the broadest possible sense: not just code, but also documentation, bug reports, forum questions (and answers), community events, new tools and plugins, and of course, new pull requests for existing tools too.
+
+---
+
+[<Image img={packagingcon} alt="Banner image for PackagingCon 2023"/>](https://packaging-con.org/)
+
+### PackagingCon 2023, October 26-28, Berlin
+
+[**PackagingCon 2023**](https://packaging-con.org/) brings together the software packaging community to present and discuss common challenges and unique solutions to software packaging. The conference is open to all packaging communities, including language-specific, OS-specific, and broader solutions such as conda.
+
+**[Abstract submission](https://cfp.packaging-con.org/2023/cfp) and [early registration](https://ti.to/packagingcon/packagingcon-2023) are now open for PackagingCon**.
+
+PackagingCon 2023 will be held October 26-28 in Berlin, and will be a hybrid event. The call for submissions closes July 31. Early registration ends September 8, *but (in-person) space is limited and you are encouraged to register early.*
+
+## Rattler!
+
+<Image img={rattler} alt="Banner image for Rattler, from GitHub project"/>
+
+**[rattler](https://github.com/mamba-org/rattler) is a new collection of Rust libraries to work with conda packages made by the [prefix.dev](https://prefix.dev/) team!**
+
+* rattler 0.3.0 just came out with lots of highlights, such as JLAP support, local `file://` urls and authenticated requests with the new `rattler_networking`. See the full ( and impressive) [changelog](https://github.com/mamba-org/rattler/releases/tag/v0.3.0) for more.
```

Comment:
```suggestion
* rattler 0.3.0 just came out with lots of highlights, such as JLAP support, local `file://` urls, and authenticated requests with the new `rattler_networking`. See the full (and impressive) [changelog](https://github.com/mamba-org/rattler/releases/tag/v0.3.0) for more.
```

Reply:
Is that `[changelog]` up there waiting for a link? I don't see one anywhere else in the file.

Reply:
I don't understand the question.  The changelog link works for me.